timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -193,30 +201,52 @@ def get_preset_models(
             f"hyperparameters must be a dict, a string or None (received {type(hyperparameters)}). "
             f"Please see the documentation for TimeSeriesPredictor.fit"
         )
+    # Handle model names ending with "Model", e.g., "DeepARModel" is mapped to "DeepAR"
+    hyperparameters_clean = {}
+    for key, value in hyperparameters.items():
+        if isinstance(key, str):
+            key = normalize_model_type_name(key)
+        hyperparameters_clean[key] = value
```

Comment:
Move to a method/function, can then add a unit test

Reply:
I'm testing that this function works correctly at the level of the trainer (`test_when_some_models_have_incorrect_suffix_then_correct_model_are_trained`). Are there some other test cases you think we should cover?

Reply:
thats fine